administrator/components/com_redirect/tmpl/links/default_addform.php

```diff
@@ -30,9 +30,9 @@
 					</div>
 					<div class="controls">
 						<input class="form-control" type="text" name="new_url" id="new_url" value="" size="50">
-					</div>
-					<div id="new_url-desc" class="form-text">
-						<?php echo Text::_('COM_REDIRECT_FIELD_NEW_URL_DESC'); ?>
+						<div id="new_url-desc" class="form-text">
+							<?php echo Text::_('COM_REDIRECT_FIELD_NEW_URL_DESC'); ?>
```

Comment:
Most similar field descriptions are wrapped in a small class
```suggestion
							<small class="form-text text-muted">
								<?php echo Text::_('COM_REDIRECT_FIELD_NEW_URL_DESC'); ?>
							</small>
```

Reply:
Per [Bootstrap Migration](https://getbootstrap.com/docs/5.0/migration/):

```
.form-text no longer sets display, but does set color and font-size. So instead of <small class="form-text text-muted"> you should now use <div class="form-text">.
```

Reply:
I will do a separate PR to remove `text-muted`.

Reply:
while small and form-text create the same size font in this template it is missing the semantic meaning that the small element gives it. So please change the div to small